components/ai_chat/ai_chat_api.cc

```diff
@@ -82,23 +84,32 @@ AIChatAPI::AIChatAPI(
 
 AIChatAPI::~AIChatAPI() = default;
 
-void AIChatAPI::QueryPrompt(ResponseCallback callback,
-                            const std::string& prompt) {
+void AIChatAPI::QueryPrompt(
```

Comment:
We still need a non-streaming QueryPrompt function, for internal queries which can have simpler callbacks, and for when streaming is not supported by the server API

Reply:
Added

Reply:
I've added the changes to make this workable in my follow-up branch

```cpp
  const bool is_sse_enabled = ai_chat::features::kAIChatSSE.Get() &&
      !data_received_callback.is_null()
```